Bottom: db41d01452eb991368fd0e375ee4801feeeb5cf3
Top:    c7e77de36b7c7cd8b0f19768cd8ae7d3af8b76b4
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-19 22:15:04 +1300

Refresh of copy-stream

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index ea4a776..ab872f8 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -388,10 +388,6 @@ class CopyStreamResult(StreamResult):
         super(CopyStreamResult, self).stopTestRun()
         domap(methodcaller('stopTestRun'), self.targets)
 
-    def estimate(self, *args, **kwargs):
-        super(CopyStreamResult, self).estimate(*args, **kwargs)
-        domap(methodcaller('estimate', *args, **kwargs), self.targets)
-
     def file(self, *args, **kwargs):
         super(CopyStreamResult, self).file(*args, **kwargs)
         domap(methodcaller('file', *args, **kwargs), self.targets)
diff --git a/testtools/tests/test_testresult.py b/testtools/tests/test_testresult.py
index 4291457..9900db5 100644
--- a/testtools/tests/test_testresult.py
+++ b/testtools/tests/test_testresult.py
@@ -586,13 +586,6 @@ class TestCopyStreamResultCopies(TestCase):
         self.assertThat(self.targets,
             AllMatch(Equals([('startTestRun',), ('stopTestRun',)])))
 
-    def test_estimate(self):
-        self.result.startTestRun()
-        now = datetime.datetime.now(utc)
-        self.result.estimate(2, route_code='abc', timestamp=now)
-        self.assertThat(self.targets,
-            AllMatch(Equals([('startTestRun',), ('estimate', 2, 'abc', now)])))
-
     def test_file(self):
         self.result.startTestRun()
         now = datetime.datetime.now(utc)
